knora-ontologies/knora-base.ttl

```diff
@@ -2730,11 +2700,7 @@
 
 :TextFileValue rdf:type owl:Class ;
```

Comment:
Are we going to store a document in the triplestore itself or are we going to store a file path?

I think in case of an XML file or any other simple plain text format we could store it in the triplestore, but not other non trivial formats.